### PR TITLE
Add a detailed HOW-TO document. Fixes #186.

### LINKED ISSUE #186: Splits docs into "quickstart", "overview" and "how-to guides"
We should have "how-to guides" for App Engine and GKE.

### PR DESCRIPTION
**Pull request by brianquinlan**

### CODE REVIEW

File: DEPLOY_NEW_HOWTO.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+   to the [Python Development Environment Setup Guide](https://cloud.google.com/python/setup).
+
+5. If you don't already one, you need to create a 
+   [Google Cloud Platform billing account](https://console.cloud.google.com/billing).
+   If you are new to Google Cloud Platform, you may be able to take advantage of
+   a [free trial](https://cloud.google.com/free/).
+
+## Types of deployment
+
+When deploying your Django application, you can choose which Google serving
```
Comment (enriquejosepadilla) [inline DEPLOY_NEW_HOWTO.md]: Is there another way to phrase Google serving tecnology?

File: DEPLOY_NEW_HOWTO.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+technology to use:
+
+### App Engine Standard Environment
```
Comment (enriquejosepadilla) [inline DEPLOY_NEW_HOWTO.md]: Is it worth mentioning Python 3?

**Comment by enriquejosepadilla** [comment]
Added some quibbles, but great documentation!

File: DEPLOY_NEW_HOWTO.md (hunk 1 above)
Comment (brianquinlan) [reply DEPLOY_NEW_HOWTO.md]: Google hosting technology? Google deployment technology?

File: DEPLOY_NEW_HOWTO.md (hunk 2 above)
Comment (brianquinlan) [reply DEPLOY_NEW_HOWTO.md]: I'm not sure what you mean here.

**Pull request merged**